src/firecracker/src/main.rs

```diff
@@ -408,7 +419,7 @@ fn main_exitable() -> FcExitCode {
     }
 }
 
-fn main() {
+fn main() -> Result<(), FcExitError> {
```

Comment:
Per https://github.com/firecracker-microvm/firecracker/pull/3575/commits/f83b7c8cd3fdb3b320ee1fe1f80769c4c498ee56#r1163830246 this should return something like `Result<(),MainExitableError>`.